desktop/apps/auction/reducers/__tests__/artworkBrowser.test.js

```diff
@@ -12,7 +12,7 @@ describe('auction/actions/artworkBrowser.test.js', () => {
         filterParams
       }
     } = auctions(undefined, {})
-    filterParams.should.containEql({ page: 1, size: 50 })
+    filterParams.should.containEql({ page: 1, size: 20 })
```

Comment:
@sweir27 - Dropped the initial fetch down a bit to speed things up, particularly on mobile. I really can't imagine needing 50 up-front when infinite scroll works so well.